kubectl-plugin/kubectl-waiops

```diff
@@ -18,6 +18,17 @@ normal=$(tput sgr0) #reset color/bolding
 CLUSTER_ADMIN=$(oc auth can-i '*' '*' --all-namespaces)
 SECURE_TUNNEL_EXISTS=""
 SUMMARY_FAILURE=""
+USER_LEVEL=$(oc whoami) # to check if the user is running the tool at higher than a basic user role
```

Comment:
Would a basic user still be able to query things within the namespace?

Reply:
Hi @kvijai82 -- no, they would not be able to query things within the namespace. For example, this is a basic-user role with `cp4waiops` and `ibm-common-services`:

![image](https://user-images.githubusercontent.com/91081503/208733410-b86acabd-970b-4af2-9621-8528c6120684.png)

They would still not be able to use any `oc` commands in either the `cp4waiops` or `ibm-common-services` project, even a simple `oc get pods -A`:

Message after `oc login` as basic-user role:
<img width="545" alt="Screenshot 2022-12-20 at 9 53 28 AM" src="https://user-images.githubusercontent.com/91081503/208733565-594e788d-e3ec-4cd0-b4d0-0cda03043890.png">

`oc project cp4waiops`:
<img width="774" alt="Screenshot 2022-12-20 at 9 53 41 AM" src="https://user-images.githubusercontent.com/91081503/208733604-296d0ef0-17e9-4de6-b123-455900d0be6f.png">